test/acceptance/tests/fixtures/bases/static-client/deployment.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
         - name: static-client
-          image: tutum/curl:latest
+          image: docker.mirror.hashicorp.services/curlimages/curl:latest
```

Comment:
This is a more up-to-date and maintained image. `tutum/curl` was so old that our mirror couldn't mirror it because it had very old metadata